<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>advancedOptions</class>
 <widget class="QMainWindow" name="advancedOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>386</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Advanced Options</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDateEdit" name="adminToAutoDateEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
    <property name="date">
     <date>
      <year>2011</year>
      <month>4</month>
      <day>12</day>
     </date>
    </property>
   </widget>
   <widget class="QTimeEdit" name="adminToAutoTimeEdit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>118</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Date</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>20</width>
      <height>321</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Time</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>20</width>
      <height>321</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>491</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>491</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>491</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>260</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Time</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="autoToAdminDateEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
    <property name="date">
     <date>
      <year>2011</year>
      <month>4</month>
      <day>12</day>
     </date>
    </property>
   </widget>
   <widget class="QTimeEdit" name="autoToAdminTimeEdit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>260</y>
      <width>118</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="setAutoToAdmin">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>310</y>
      <width>176</width>
      <height>27</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="autoToAdmin">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Date and Time to Switch from Automatic Mode to Admin mode</string>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="setAdminToAuto">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>176</width>
      <height>27</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="adminToAuto">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Date and Time to Switch from Admin Mode to Automatic mode</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
